reaktive/src/nativeCommonTest/kotlin/com/badoo/reaktive/observable/ToListTestNative.kt

```diff
@@ -0,0 +1,24 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.test.observable.TestObservable
+import com.badoo.reaktive.test.observable.onNext
+import com.badoo.reaktive.test.single.assertSuccess
+import com.badoo.reaktive.test.single.test
+import kotlin.native.concurrent.isFrozen
+import kotlin.test.Test
+import kotlin.test.assertFalse
+
+class ToListTestNative {
+
+    @Test
+    fun collects_all_items_without_freezing() {
+        val upstream = TestObservable<Int?>(autoFreeze = false)
+        val observer = upstream.toList().test(autoFreeze = false)
+
+        upstream.onNext(0, null, 1, null, 2)
+        upstream.onComplete()
+
+        observer.assertSuccess(listOf(0, null, 1, null, 2))
+        assertFalse(observer.value.isFrozen)
```

Comment:
Do we need `isFrozen` as part of our util package and available for all platforms? Like `freeze()` or `ensureNeverFrozen()`? Just return `false` for all platforms except native.

In this case we will have single test implementation for all platforms.